pkg/engine/factories/contextloaderfactory.go

```diff
@@ -85,30 +85,30 @@ func (l *contextLoader) newLoader(
 	if entry.ConfigMap != nil {
 		if l.cmResolver != nil {
 			l := loaders.NewConfigMapLoader(ctx, l.logger, entry, l.cmResolver, jsonContext)
-			return enginecontext.NewDeferredLoader(entry.Name, l)
+			return enginecontext.NewDeferredLoader(entry.Name, l, logr.Logger{})
 		} else {
 			l.logger.Info("disabled loading of ConfigMap context entry %s", entry.Name)
 			return nil, nil
 		}
 	} else if entry.APICall != nil {
 		if client != nil {
 			l := loaders.NewAPILoader(ctx, l.logger, entry, jsonContext, jp, client)
```

Comment:
You can rename the `l` in `l := loaders.NewAPILoader(...` to something like `ldr`.